<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="useCodeLabel">
      <property name="text">
       <string>有效验证码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="confirmBtn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="macLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="countLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="useCodeLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="macLabel">
      <property name="text">
       <string>MAC地址：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="copyBtn">
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>登陆</string>
    </property>
    <addaction name="actionpassword"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionpassword">
   <property name="text">
    <string>输入密码</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
